src/app/containers/MediaPlayer/index.jsx

```diff
@@ -72,6 +72,10 @@ const MediaPlayerContainer = ({
       aresMediaBlock,
     ),
     type: format === 'audio' ? 'audio' : 'video',
+    guidanceMessage: path(
+      ['model', 'blocks', 0, 'model', 'versions', 0, 'warnings', 'short'],
```

Comment:
What does this return when the `warnings` object is empty? (as this is the standard when media has no warnings)

Reply:
we should Return `null` when the object is empty, Reverting back to pathOr to enable this

Reply:
`guidanceMessage` shouldn't require setting to `null` as that's it's default value. See: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-media-player/src/Placeholder/index.jsx#L81